drivers/espi/espi_it8xxx2.c

```diff
@@ -153,6 +155,97 @@ static const struct ec2i_t pmc1_settings[] = {
 	{HOST_INDEX_LDA, 0x01},
 };
 
+#ifdef CONFIG_ESPI_PERIPHERAL_EC_HOST_CMD
+#define IT8XXX2_ESPI_HC_DATA_PORT_MSB \
+	((CONFIG_ESPI_PERIPHERAL_HOST_CMD_DATA_PORT_NUM >> 8) & 0xff)
+#define IT8XXX2_ESPI_HC_DATA_PORT_LSB \
+	(CONFIG_ESPI_PERIPHERAL_HOST_CMD_DATA_PORT_NUM & 0xff)
+#define IT8XXX2_ESPI_HC_CMD_PORT_MSB \
+	(((CONFIG_ESPI_PERIPHERAL_HOST_CMD_DATA_PORT_NUM + 4) >> 8) & 0xff)
+#define IT8XXX2_ESPI_HC_CMD_PORT_LSB \
+	((CONFIG_ESPI_PERIPHERAL_HOST_CMD_DATA_PORT_NUM + 4) & 0xff)
+static const struct ec2i_t pmc2_settings[] = {
+	/* Select logical device 12h(PM2 host command) */
+	{HOST_INDEX_LDN, LDN_PMC2},
+	/* I/O Port Base Address (data/command ports) */
+	{HOST_INDEX_IOBAD0_MSB, IT8XXX2_ESPI_HC_DATA_PORT_MSB},
+	{HOST_INDEX_IOBAD0_LSB, IT8XXX2_ESPI_HC_DATA_PORT_LSB},
+	{HOST_INDEX_IOBAD1_MSB, IT8XXX2_ESPI_HC_CMD_PORT_MSB},
+	{HOST_INDEX_IOBAD1_LSB, IT8XXX2_ESPI_HC_CMD_PORT_LSB},
+	/* Set IRQ=00h for logical device */
+	{HOST_INDEX_IRQNUMX, 0x00},
+	/* Enable logical device */
+	{HOST_INDEX_LDA, 0x01},
+};
+#endif
+
+#if defined(CONFIG_ESPI_PERIPHERAL_EC_HOST_CMD) || \
+	defined(CONFIG_ESPI_PERIPHERAL_ACPI_SHM_REGION)
+/* Host to RAM (H2RAM) memory mapping, 4K aligned */
+#define IT8XXX2_ESPI_H2RAM_POOL_SIZE 0x1000
+static uint8_t h2ram_pool[IT8XXX2_ESPI_H2RAM_POOL_SIZE] __aligned(0x1000);
```

Comment:
The cros-ec driver allocated this pool in a separate linker section so it was always at a fixed address.   Allocating this into the .bss section could potentially waste up to 4 KiB of RAM due to the alignment requirement.

Reply:
Putting this pool inside the first section of the RAM should be able to avoid gap. Can we fix this symptom later?  We need to add a location where is at the beginning of the RAM to zephyr_linker_sources().

Reply:
I would prefer to see this gap avoided and not wait until later to fix the symptom

Reply:
RAM size is already low on the Hayato build (86% used).  So please address the alignment issue.

Reply:
Done.
The first one commit is required to avoid the gap.